runtime/plugin/netrwPlugin.vim

```diff
@@ -31,6 +32,67 @@ set cpo&vim
 " ---------------------------------------------------------------------
 " Public Interface: {{{1
 
+" Commands Launch/URL {{{2
+" surpress output of command; use bang for GUI applications
+
+" set up redirection (avoids browser messages)
+" by default, g:netrw_suppress_gx_mesg is true
+if get (g:, ':netrw_suppress_gx_mesg', 1)
+  if &srr =~# "%s"
+    if has("win32")
+      let s:redir= substitute(&srr,"%s","nul","")
+    else
+      let s:redir= substitute(&srr,"%s","/dev/null","")
+    endif
+  else
+    if has("win32")
+      let s:redir= &srr . "nul"
+    else
+      let s:redir= &srr . "/dev/null"
+    endif
+  endif
+else
+  let s:redir= ""
+endif
+"  call Decho("set up redirection: redir{".redir."} srr{".&srr."}",'~'.expand("<slnum>"))
+
+if has('unix')
+  if has('win32unix') " Git Bash provides /usr/bin/start script calling cmd.exe //c
+    " use start //b "" to set void title and avoid ambiguity with passed argument
+    silent! command -complete=shellcmd -nargs=1 -bang Launch
```

Comment:
See https://github.com/vim/vim/issues/14872 for examples; to read any output of input in Vim, say binary, or even text, like a web page.

Reply:
Yes, I don't think that it isn't useful. I hardly imagine a situation where a user wants a command to launch an app. I think this is more useful as a function for scripting

Reply:
I think this is current practice. At the same time, there are so many plug-ins whose sole raison d'etre is launching some command, say https://github.com/romainl/vim-devdocs, https://github.com/wolandark/vim-live-server/, various markdown live preview [plug-ins](https://github.com/arminveres/md-pdf.nvim/issues/19) vs. launching [Okular](https://gist.github.com/Konfekt/c2066abc31b41430417310bd6e6d795e). Similar to using `:compiler` instead of installing linter or compiler plug-ins, one could opt for `:Launch` instead of installing application specific plug-ins.

Reply:
Why the `:silent!`?

Reply:
Thank you. Since there's the `g:loaded_netrwPlugin` load guard, I'd say it's dispensable and dropped it.